modules/ROOT/pages/deployment-strategies.adoc

```diff
@@ -83,8 +83,17 @@ The hybrid scenario of hosting your apps on your own servers while managing them
 
 To make a server visible to Runtime Manager, you must first register it by running an instruction on the server's command line, see xref:managing-servers.adoc#add-a-server[Add a Server]. You then have the flexibility to group servers into xref:managing-servers.adoc#create-a-server-group[Server Groups] or xref:managing-servers.adoc#create-a-cluster[Clusters] to make them a <<High Availability>> deployment target. You can then choose to xref:deploying-to-your-own-servers.adoc[deploy your applications] to either a server, server group or cluster.
 
-[TIP]
-See the sections below to find out what specific operating features apply to this modality and how to configure them.
+==== Security and Hybrid Deployment 
+
+Unless you choose to configure it otherwise, no application data flows to the cloud, only metadata pushed by the Mule Runtime agent in each Mule runtime engine.
```

Comment:
what does data flows stand for in here? - adding "default" might make more sense.